index.ts

```diff
@@ -46,14 +50,14 @@ export const recommendationsToHTML = (
 ) => `
 <div class="${classes.wrapper}">
 ${recommendations.reduce(
-  (output, { title, description, imageUri }) =>
+  (output, { title, description, uri, imageUri }) =>
     `${output}
     
-  <div class="${classes.card}">
+  <a href="${uri}" class="${classes.card}">
```

Comment:
Hmm, should it instead be `<a><div></div></a>`?

Reply:
Idk. I was looking at https://stackoverflow.com/questions/49554070/using-bootstrap-cards-as-a-hyperlink which seemed to have the following suggestions/options:
1. Replace the card `<div>` with `<a>`. Might have to make `a` be `display: block` if certain CSS behavior is needed but that should be fine.
2. Wrap the `<div>` in a `<a>`. (Your suggestion)
3. Add `<a>` as a child of the `<div>` and use `position: absolute` to cover the whole card.

Number one seemed to be the simplest HTML and there was no second card wrapper in the HTML or that you might have to style separately. But if there's a reason to do another one if them, that's fine with me. I haven't written any CSS in this repo and was going to let the consumer style everything, so I guess whatever is most flexible yet simple would be good. But maybe there's a reason to write some CSS in here too.

Reply:
At least on the AEM side of things, wrapping the `<div>` in an `<a>` tag is the least amount of work.

Reply:
@j2trumpet any opinions on what might be better on the wordpress side?